packages/authentication/android/src/main/java/io/capawesome/capacitorjs/plugins/firebase/authentication/FirebaseAuthentication.java

```diff
@@ -587,6 +587,16 @@ public void updateEmail(FirebaseUser user, @NonNull String newEmail, @NonNull Ru
             );
     }
 
+    public void verifyBeforeUpdateEmail(FirebaseUser user, @NonNull String newEmail, @NonNull ActionCodeSettings actionCodeSettings, @NonNull Runnable callback) {
+        user
+            .verifyBeforeUpdateEmail(newEmail, actionCodeSettings)
+            .addOnCompleteListener(
+                task -> {
+                    callback.run();
+                }
+            );
```

Comment:
```suggestion
            .addOnSuccessListener(unused -> callback.success())
            .addOnFailureListener(exception -> callback.error(exception));
```

I guess this method can throw an excpetion or not? If so, then we should handle this.

Reply:
Thanks for pointing that out. Feel free to create a new PR for the other methods. For now, just push that single change.